src/components/Modal/Modal.module.scss

```diff
@@ -0,0 +1,23 @@
+.Overlay {
+    display: grid;
+    position: fixed;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+
+    place-items: center;
+    overflow-y: auto;
+    z-index: 10000;
```

Comment:
I recommend to use some variable for z-index something like `$z-index-100~10000` or  `$z-index-xxs~xxl`

Reply:
I think in this case we wouldn't want anything to share the same zIndex as the `Modal` component, as it's meant to be on top. Let me know if this assumption is incorrect though!